packages/schema-generator/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './generate_schema.js';
```

Comment:
nit: do we need the `.js` extension needed here?

Reply:
Yes, it is required. That's the style used in the repo.